staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -3745,13 +3745,38 @@ func describeNetworkPolicySpec(nps networkingv1.NetworkPolicySpec, w PrefixWrite
 	} else {
 		w.Write(LEVEL_2, "%s\n", metav1.FormatLabelSelector(&nps.PodSelector))
 	}
-	w.Write(LEVEL_1, "Allowing ingress traffic:\n")
-	printNetworkPolicySpecIngressFrom(nps.Ingress, "    ", w)
-	w.Write(LEVEL_1, "Allowing egress traffic:\n")
-	printNetworkPolicySpecEgressTo(nps.Egress, "    ", w)
+
+	ingressEnabled, egressEnabled := getPolicyType(nps)
+	if ingressEnabled {
+		w.Write(LEVEL_1, "Allowing ingress traffic:\n")
+		printNetworkPolicySpecIngressFrom(nps.Ingress, "    ", w)
+	} else {
+		w.Write(LEVEL_1, "Allowing all ingress traffic\n")
```

Comment:
That's not right; it should say "Doesn't affect ingress traffic" or something like that. If ingress traffic would have been allowed without this policy, then it's still allowed with this policy, and if it would have been denied without this policy, then it's still denied with this policy. (Likewise for egress.)

Reply:
Yeah I agree with that, will fix now